css/error.css

```diff
@@ -0,0 +1,53 @@
+.container {
+  align-content: center;
+  padding-top: 5%;
+  width: 75%; }
+  @media (min-width: 1600px) {
+    .container {
+      padding-top: 10%; } }
+
+.error-information {
+  text-align: center;
+  padding-top: 5%;
+  background: rgba(255, 255, 255, 0.75);
+  padding: 3%;
+  border-radius: 33px; }
+  .error-information #error-number {
+    font-size: 300%; }
+
+.logo {
+  margin-left: 30%;
+  margin-right: 50%;
+  margin-bottom: 7%; }
+  .logo #technica-logo {
+    display: block;
+    width: 15vw;
+    height: 4vw; }
+    @media (max-width: 1500px) {
+      .logo #technica-logo {
+        width: 60vw;
+        height: 15vw;
+        margin-left: -100%; } }
```

Comment:
Technica logo size & centering should be good on all devices smaller than my 13" laptop dimensions - can you add media queries to match the proportions & center on all bigger screen sizes?